### PR TITLE
fix: update tf landing hero img

### PR DESCRIPTION
**Pull request by zchsh**
## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Restores image update made in #972 , this time ensure there's top padding built into the image (which is why we replaced the previously update-text image in #978).

## 🤷 Why

To use image approved by education ([Slack thread][slack]).

## 🛠️ How

Updates Dato URL.

## 📸 Design Screenshots

CleanShot 2022-09-13 at 17 48 47@2x

## 🧪 Testing

- [ ] Visit [/terraform][preview], the landing hero image should read `backend-prod`

[task]: https://app.asana.com/0/0/1202975892434092/f
[preview]: https://www.hashicorp.com
[slack]: https://hashicorp.slack.com/archives/C01KCU4HDPY/p1662748999069179?thread_ts=1662746135.472499&cid=C01KCU4HDPY

### CODE REVIEW

**Review by kendallstrautman — APPROVED** [review]
My bad!

**Comment by mhonnatti** [comment]
@zchsh any concerns with the subtle differences between `backend-prod` image and the `user-feedback-prod` image?
- purple lines in the background are oriented differently
- no "Latest Run"
- Terraform logo is slightly different

If we have to wait for Brayden to do anything, then I'm happy to approve now and change the asset later. LMK

**Comment by zchsh** [comment]
@kendallstrautman all good, the padding on the old image i’d uploaded was off 😅 

@mhonnatti I’m not 100% sure on the impact of those changes, personally i don’t have concerns. Mostly going off the approval from Judith on #972 (i’d made a small text edit thag Judith requested, and re-exported assets from the approved spec in Figma since design was out).

if we want to go back and re-do the asset with exact patterns etc, i’d definitely leave that up to design, as I don’t have the Figma edit access I used to and am not sure where the canonical export file (i assumed it was the Terraform page in the Developer file 😬).

merging this is for now, we can definitely adjust if we need to tweak the logo size, pattern, etc. 👍 cc @braydenlove

**Pull request merged**
